### DISCUSSION TITLE
Add Glowstone dust /Compact

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [EssentialsX/Essentials#4300] [Ideas and Feature Suggestions] [answered]
The "/compress" came to facilitate the formation of blocks, if it had as with glowstone it would be nice

**Comment by pop4959** [comment, marked as answer]
compress only works with reversible recipes

**Reply by triagonal** [reply]
See https://github.com/EssentialsX/Essentials/issues/2718 for more info.
